src/compute/src/server.rs

```diff
@@ -325,3 +327,16 @@ pub async fn compute_node_serve(
 
     (join_handle_vec, shutdown_send)
 }
+
+/// Check whether the compute node has enough memory to perform computing tasks. Apart from storage,
+/// it must reserve at least `MIN_COMPUTE_MEMORY_MB` for computing. Otherwise, it is not allowed to
+/// start.
+fn validate_compute_node_memory_config(total_memory_bytes: usize, storage_config: &StorageConfig) {
+    if storage_config.total_memory_limit_mb() << 20 > total_memory_bytes {
```

Comment:
Can we refactor these `<< 20` in some utility methods, for example `mb_to_bytes`?

Reply:
There's really a looot of `<< 20` in our code. Maybe we can do the refactor for all MB-to-bytes later rather than in this PR 🤣